dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/AbstractUniqueObjectController.java

```diff
@@ -126,4 +137,23 @@ protected U buildExampleWithUuid(UUID uuid) throws ServiceException {
           "Cannot construct example " + objectType + " for uuid=" + uuid + ": " + e, e);
     }
   }
+
+  protected List<U> buildExamplesWithUuids(UUID[] uuids) throws ServiceException {
```

Comment:
Dann kann mans mit einer oder mehreren UUIDs aufrufen oder gar ein `UUID[]` übergeben. Der Parameter `uuids` in der Methode ist immer ein Array (nie null).

Dann könnte man auch die Logik von `buildExampleWithUuid` hier unterbringen und sie stattdessen auf diese Methode "verweisen" lassen.
```suggestion
  protected List<U> buildExamplesWithUuids(UUID... uuids) throws ServiceException {
```

Reply:
Eher ungerne, denn eine Varargs sind auf 65k längenbegrenzt, und eine UUID benötigt 16 Bytes, d.h. man könnte maximal 4K UUIDs übergeben, bevor es zu einem Speicherfehler kommt

Reply:
Ah, das wusste ich nicht. Aber ich denke, dass man bei 4000 UUIDs eher in einen Timeout läuft.